dotnet-client-libraries/BenchmarkDotNetWithUnitTests/Tests/BenchmarkFixture.cs

```diff
@@ -0,0 +1,32 @@
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Order;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNetWithUnitTests;
+
+namespace Tests;
+
+public class BenchmarkFixture : IDisposable
```

Comment:
There is nothing in the class that is of IDisposable, so there isn't a need to implement the interface.